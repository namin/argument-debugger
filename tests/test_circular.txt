The Bible is true because it's the word of God.
We know it's the word of God because the Bible says so.
Therefore, we should follow the Bible.
