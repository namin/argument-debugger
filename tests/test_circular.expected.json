{
  "must_find": ["circular"],
  "may_find": ["unsupported_premise", "missing_link"],
  "must_not_find": ["false_dichotomy"]
}